pom.xml

```diff
@@ -54,13 +54,21 @@
 
     <properties>
         <sling.java.version>8</sling.java.version>
+        <java.locale.providers>COMPAT,JRE,SPI</java.locale.providers>
```

Comment:
Would you mind adding a comment that CLDR changed drastically between JDK8 and 11?

Reply:
done